modules/core/02-client/genesis.go

```diff
@@ -59,6 +59,8 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, gs types.GenesisState) {
 }
 
 // ExportGenesis returns the ibc client submodule's exported genesis.
+// NOTE: the export process is not optimized, it will iterate three
+// times over the 02-client sub-store.
```

Comment:
I thought about fixing this, but it started getting too complicated, so I decided it wasn't worth it

Reply:
yup definitely!